sim/druid/item_sets_pve_phase_5.go

```diff
@@ -237,55 +236,76 @@ var ItemSetFuryOfStormrage = core.NewItemSet(core.ItemSet{
 
 // Swipe(Bear) also causes your Maul to hit 1 additional target for the next 6 sec.
 func (druid *Druid) applyT2Guardian2PBonus() {
-	label := "S03 - Item - T2 - Druid - Guardian 2P Bonus"
-	if druid.HasAura(label) {
+	if druid.Env.GetNumTargets() == 1 {
 		return
 	}
```

Comment:
We'll also want to keep this aura check but it can go after the target count check, that way we don't double dip (I think it would just error from the same spell registered twice)